cli/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/hashicorp/consul-k8s/cli
 
-go 1.22
+go 1.22.0
 
-toolchain go1.22.5
+toolchain go1.22.6
```

Comment:
did we mean to bump the toolchain here? it's probably good that we do (and should probably be on the latest minor version tbh)

Reply:
I did not mean to -- I must be running a newer version of Go on my machine than what we use in the repo.

I'll bump it back down just to avoid sprawl in this PR